docs/monthday.md

```diff
@@ -325,15 +325,15 @@ Object.assign({}, md).day  // => undefined
 Object.assign({}, md.getFields()).day  // => 24
 ```
 
-### monthDay.**getISOFields**(): { refISOYear: number, isoMonth: number, isoDay: number }
+### monthDay.**getISOFields**(): { isoYear: number, isoMonth: number, isoDay: number, calendar: object }
 
 **Returns:** a plain object with properties expressing `monthDay` in the ISO 8601 calendar, as well as the value of `monthDay.calendar`.
 
 This method is mainly useful if you are implementing a custom calendar.
 Most code will not need to use it.
 Use `monthDay.getFields()` instead.
 
-The value of the `refISOYear` property will be equal to the `refISOYear` constructor argument passed when `monthDay` was constructed.
+The value of the `isoYear` property will be equal to the `refISOYear` constructor argument passed when `monthDay` was constructed.
```

Comment:
Why not just change the name everywhere instead of keeping `refISOYear` as the constructor argument name?

Reply:
I did find it useful to keep "ref" in the name of the constructor argument, because there I think it's most important to make clear that it is for reference only.

Reply:
Since brevity is no longer a concern, since it's only the name of the argument in the constructor, how about naming it "referenceISOYear" to make it even more clear?